website/pages/guides/[[...page]].jsx

```diff
@@ -10,15 +10,12 @@ import {
 const BASE_ROUTE = 'guides'
 const NAV_DATA = 'data/guides-nav-data.json'
 const CONTENT_DIR = 'content/guides'
+const MAIN_BRANCH = 'master'
+const PRODUCT = { name: productName, slug: productSlug }
 
 export default function GuidesLayout(props) {
   return (
-    <DocsPage
-      baseRoute={BASE_ROUTE}
-      mainBranch="master" // used for "edit on this page", default "main"
-      product={{ name: productName, slug: productSlug }}
-      staticProps={props}
-    />
+    <DocsPage baseRoute={BASE_ROUTE} product={PRODUCT} staticProps={props} />
```

Comment:
Changes to the `docs`, `guides`, and `intro` page routes reflect the slightly revised API in the stable release of `docs-page`.

Reply:
Was the `mainBranch` prop removed on purpose? Just want to double check

Reply:
It was on purpose! Previously `docs-page` was using it to construct the GitHub `Edit this page` link, but that was all. Now with the flexibility needed to construct a GitHub edit link not based on branch but instead on version, it felt logical to switch up the interface. (see also [related comment on react-components PR](https://github.com/hashicorp/react-components/pull/154/files#r588828157)